src/utils/utils.ts

```diff
@@ -73,3 +87,36 @@ export const calculateTimeRemaining = (startDate: Date) => {
     seconds: seconds % 60,
   };
 };
+
+/**
+ * Returns whether the given date is in the range of a given start and end date.
+ *
+ * @param startDate - A date, must be before [endDate].
+ * @param endDate - A date, must be after [startDate].
+ * @param targetDate - Optional. The date that is to be checked if it's in range of [startDate] and [endDate]. If not provided, the current date and time when the function is called will be used.
+ * @returns A boolean stating if the current date is greater than or equal to startDate and less than or equal to endDate.
+ */
+export const dateInRange = (
+  startDate: Date,
+  endDate: Date,
+  targetDate: Date = new Date()
+) => {
+  return targetDate >= startDate && targetDate <= endDate;
+};
+
+/**
+ * Formats a Date object into a string in the M/D 00:00 AM/PM format.
+ *
+ * @param date - A date.
+ * @returns A string representing [date] in the above format.
+ */
+export const formatDate = (date: Date) => {
```

Comment:
We may need to change the name of this function in the future, but it's fine for now.